extension.meta.xml

```diff
@@ -14,6 +14,9 @@
 		</author>
 	</authors>
 	<releases>
+		<release version="1.7.2" date="2017-05-19" min="2.5.x" max="4.x.x" php-min="5.6.x" php-max="7.x.x">
```

Comment:
That's why the PR is not merged ;) It's a changed @alexnantel88 added in her 4.0 experiment. But it is soooo nice, that I'd like to add it for real. I'll check with here if she can redo the PR, but without any of the "experimental" stuff. Thanks for the review tho ;)